server_platform_requirements.adoc

```diff
@@ -154,13 +154,15 @@ Security requirements straddle hardware and firmware.
 |===
 | ID#      ^| Requirement
 | `SEC_010`  | MUST implement UEFI Secure Boot and Driver Signing (cite:[UEFI] Section 32)
-| `SEC_011`  | It MUST be possible for a physically present user to disable Secure Boot enforcement, thus allowing unsigned code to be executed.
-| `SEC_012`  | It MUST be possible for a physically present user to fully manage the contents of all Secure Boot key stores (PK, KEK, db and dbx). This includes the ability to delete all factory-provided keys, enrolling their own custom keys, and resetting all key stores to their factory state.
+| `SEC_011`  | It MUST be possible for authorized users to disable Secure Boot enforcement, thus allowing unsigned code to be executed.
+2+| _The determination of the definition of an authorized user is left to the
+    platform integrator and customer._
+| `SEC_012`  | It MUST be possible for authorized users to fully manage the contents of all Secure Boot key stores (PK, KEK, db and dbx). This includes the ability to delete all factory-provided keys, enrolling their own custom keys, and resetting all key stores to their factory state.
 | `SEC_020`  | MUST back the UEFI Authenticated Variables implementation with
              a mechanism that cannot be accessed or tampered by an unauthorized
              software or hardware agent.
 | `SEC_030`  | MUST implement in-band firmare updates as per cite:[BRS].
 | `SEC_040`  | Firmware update payloads MUST be digitally signed.
 | `SEC_050`  | Firmware update signatures MUST be validated before being applied.
-| `SEC_060`  | It MUST not be possible to bypass secure boot, authentication or digital signature failures.
+| `SEC_060`  | It MUST not be possible for unauthorized users to bypass secure boot, authentication or digital signature failures.
```

Comment:
I think this requirement is implicit for any security mechanism which does X. I do not think you need to state the negative case.

Reply:
Hi @rsahita , is this a suggestion to remove SEC_060 entirely? Or to edit it in a different way?